README.md

```diff
@@ -25,6 +25,8 @@ git clone https://github.com/jrconlin/topics.git
 cd topics
 virtualenv .
 bin/activate
+pip install -r requirements.txt
+python setup.py develop
```

Comment:
`setup.py` should bring in the requirements automatically. I'll retest on the mini to see if there's something I need to fix up there.